apps/els_lsp/src/els_compiler_diagnostics.erl

```diff
@@ -680,18 +680,23 @@ inclusion_range(IncludePath, Document, include) ->
     [Range || #{id := Id, range := Range} <- POIs, Id =:= IncludeId];
 inclusion_range(IncludePath, Document, include_lib) ->
     POIs = els_dt_document:pois(Document, [include_lib]),
-    IncludeId = els_utils:include_lib_id(IncludePath),
+    IncludeId = els_utils:include_lib_id(absolute_path(IncludePath)),
```

Comment:
Do we know that IncludePath is always relative?

Reply:
It can also be an absolute one, but the join function takes care of that, if that's the case (maybe I should have called it `ensure_absolute_path`):

```
{ok,"/Users/robertoaloi/git/github/erlang-ls/erlang_ls"}
2> filename:join(Cwd, "/tmp/foo/bar").
"/tmp/foo/bar"
```